api/v3/Contact.php

```diff
@@ -817,10 +836,13 @@ function civicrm_api3_contact_getquick($params) {
         break;
 
       default:
-        if ($value != 'id') {
+        if ($value == 'id') {
+          $actualSelectElements[] = 'cc.id';
+        }
+        elseif ($value != 'sort_name') {
           $suffix = 'cc';
-          if (!empty($params['field_name']) && $params['field_name'] == 'value') {
-            $suffix = CRM_Utils_String::munge(CRM_Utils_Array::value('table_name', $params, 'cc'));
+          if ($field_name == $value) {
+            $suffix = $table_name;
```

Comment:
what is the impact of losing the munging?

Reply:
`$table_name` already gets munged up top (2nd line of the fn)